openassessment/xblock/test/test_rubric_reuse_mixin.py

```diff
@@ -0,0 +1,162 @@
+""" Unit tests for rubric_reuse_mixin.py """
```

Comment:
This is a nit from other files's naming convention, I think this should name `test_rubric_reuse.py`